### PR TITLE
Add iree/samples/models/simple_abs.mlir and update docs to use it.

### PR DESCRIPTION
**Pull request by ScottTodd**
The `docs/developers/get_started/...` files should be deleted and/or merged into https://github.com/google/iree/tree/main/docs/website/docs/building-from-source, so I don't care much about them...

### CODE REVIEW

File: docs/developers/get_started/getting_started_windows_bazel.md [hunk 1]
```diff
@@ -118,11 +118,11 @@ Check out what was built:
 ```
 
 Translate a
-[MLIR file](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-mlir.mlir)
+[MLIR file](https://github.com/google/iree/blob/main/iree/samples/models/simple_abs.mlir)
 and execute a function in the compiled module:
 
 ```powershell
-> .\bazel-bin\iree\tools\iree-run-mlir.exe .\iree\tools\test\iree-run-mlir.mlir -function-input="i32=-2" -iree-hal-target-backends=vmvx -print-mlir
+> .\bazel-bin\iree\tools\iree-run-mlir.exe .\iree\samples\models\simple_abs.mlir -function-input="i32=-2" -iree-hal-target-backends=vmvx -print-mlir
```
Comment (hcindyl) [inline docs/developers/get_started/getting_started_windows_bazel.md]:
```suggestion
> .\bazel-bin\iree\tools\iree-run-mlir.exe .\iree\samples\models\simple_abs.mlir -function-input="f32=-2" -iree-hal-target-backends=vmvx -print-mlir
```

File: docs/developers/get_started/getting_started_macos_cmake.md [hunk 2]
```diff
@@ -99,11 +99,11 @@ $ ../iree-build/iree/tools/iree-translate --help
...
 
 Translate a
-[MLIR file](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-mlir.mlir)
+[MLIR file](https://github.com/google/iree/blob/main/iree/samples/models/simple_abs.mlir)
 and execute a function in the compiled module:
 
 ```shell
-$ ../iree-build/iree/tools/iree-run-mlir $PWD/iree/tools/test/iree-run-mlir.mlir \
+$ ../iree-build/iree/tools/iree-run-mlir $PWD/iree/samples/models/simple_abs.mlir \
   -function-input="i32=-2" -iree-hal-target-backends=vmvx -print-mlir
```
Comment (hcindyl) [inline docs/developers/get_started/getting_started_macos_cmake.md]:
```suggestion
  -function-input="f32=-2" -iree-hal-target-backends=vmvx -print-mlir
```

File: docs/developers/get_started/getting_started_macos_bazel.md [hunk 3]
```diff
@@ -116,11 +116,11 @@ $ ./bazel-bin/iree/tools/iree-translate --help
...
 
 Translate a
-[MLIR file](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-mlir.mlir)
+[MLIR file](https://github.com/google/iree/blob/main/iree/samples/models/simple_abs.mlir)
 and execute a function in the compiled module:
 
 ```shell
-$ ./bazel-bin/iree/tools/iree-run-mlir ./iree/tools/test/iree-run-mlir.mlir \
+$ ./bazel-bin/iree/tools/iree-run-mlir ./iree/samples/models/simple_abs.mlir \
   -function-input="i32=-2" -iree-hal-target-backends=vmvx -print-mlir
```
Comment (hcindyl) [inline docs/developers/get_started/getting_started_macos_bazel.md]:
```suggestion
  -function-input="f32=-2" -iree-hal-target-backends=vmvx -print-mlir
```

File: docs/developers/get_started/getting_started_linux_bazel.md [hunk 4]
```diff
@@ -113,11 +113,11 @@ $ ./bazel-bin/iree/tools/iree-translate --help
...
 
 Translate a
-[MLIR file](https://github.com/google/iree/blob/main/iree/tools/test/iree-run-mlir.mlir)
+[MLIR file](https://github.com/google/iree/blob/main/iree/samples/models/simple_abs.mlir)
 and execute a function in the compiled module:
 
 ```shell
-$ ./bazel-bin/iree/tools/iree-run-mlir ./iree/tools/test/iree-run-mlir.mlir \
+$ ./bazel-bin/iree/tools/iree-run-mlir ./iree/samples/models/simple_abs.mlir \
   -function-input="i32=-2" -iree-hal-target-backends=vmvx -print-mlir
```
Comment (hcindyl) [inline docs/developers/get_started/getting_started_linux_bazel.md]:
```suggestion
  -function-input="f32=-2" -iree-hal-target-backends=vmvx -print-mlir
```

File: docs/developers/developing_iree/benchmarking.md [hunk 5]
```diff
@@ -20,10 +20,9 @@ To use `iree-benchmark-module`, generate an IREE module for the target backend:
 
 ```shell
 $ bazel run //iree/tools:iree-translate -- \
-  -iree-input-type=mhlo \
   -iree-mlir-to-vm-bytecode-module \
   -iree-hal-target-backends=vmvx \
-  $PWD/iree/tools/test/iree-benchmark-module.mlir \
+  $PWD/iree/samples/models/simple_abs.mlir \
```
Comment (hcindyl) [inline docs/developers/developing_iree/benchmarking.md:25]: Please also update the function input at line 36

**Review by hcindyl — CHANGES_REQUESTED** [review]
Thanks for helping out the cleanup. Please also update some corresponding function inputs.

**Comment by ScottTodd** [comment]
Whoops, thanks for catching those. Updated.

**Review by hcindyl — APPROVED** [review]

**Pull request merged**